docker-compose.arm64.yml

```diff
@@ -3,15 +3,15 @@ services:
     gateway:
         ports:
             - 8080:8080
-        image: openfaas/gateway:0.10.1-arm64
+        image: openfaas/gateway:0.10.2-arm64
```

Comment:
I'm sorry I can't merge a config change before the images exist, and I can't get the images to exist before merging the changes that create the Makefiles. Can you please work more atomically and split into two separate PRs?

Reply:
I reverted back to the current Gateway tag. 
I'll split the PR in two - one for the Makefiles and one for bumping the versions after the images are built.